### DISCUSSION TITLE
What is nuclei cloud / how to get nuclei cloud API key?

### DISCUSSION THREAD

**Discussion by qianbenhyu** [projectdiscovery/nuclei#2755] [Q&A] [answered]
as it shows, how to gen/get a cloud api-key?
![image](https://user-images.githubusercontent.com/32768810/197482491-2c266ea7-0485-40fa-9177-cac0157b3fea.png)

**Comment by ehsandeep** [comment, marked as answer]
Hey @qianbenhyu, those options added for internal dev / testing and not available to use yet, we will share more details later when it's available.

**Reply by jahithoque** [reply]
It's been 2 years now. Shared the details?

**Reply by ehsandeep** [reply]
https://docs.projectdiscovery.io/cloud/scanning/nuclei-scan#set-up-your-api-key

**Comment by qianbenhyu** [comment]
I get it, thanks!

**Comment by ehsandeep** [comment]
See https://docs.projectdiscovery.io/cloud/scanning/nuclei-scan#set-up-your-api-key
